crates/bws/src/command/project.rs

```diff
@@ -0,0 +1,119 @@
+        ProjectCreateRequest, ProjectGetRequest, ProjectPutRequest, ProjectsDeleteRequest,
+        ProjectsListRequest,
+    },
+    Client,
+};
+use color_eyre::eyre::Result;
+use uuid::Uuid;
+
+use super::OutputSettings;
+use crate::render::serialize_response;
+
+pub(crate) async fn list(
+    mut client: Client,
+    organization_id: Uuid,
+    output_settings: OutputSettings,
+) -> Result<()> {
+    let projects = client
+        .projects()
+        .list(&ProjectsListRequest { organization_id })
+        .await?
+        .data;
+    serialize_response(projects, output_settings.output, output_settings.color);
+
+    Ok(())
+}
+
+pub(crate) async fn get(
+    mut client: Client,
+    project_id: Uuid,
+    output_settings: OutputSettings,
+) -> Result<()> {
+    let project = client
+        .projects()
+        .get(&ProjectGetRequest { id: project_id })
+        .await?;
+    serialize_response(project, output_settings.output, output_settings.color);
+
+    Ok(())
+}
+
+pub(crate) async fn create(
+    mut client: Client,
+    organization_id: Uuid,
+    name: String,
+    output_settings: OutputSettings,
+) -> Result<()> {
+    let project = client
+        .projects()
+        .create(&ProjectCreateRequest {
+            organization_id,
+            name,
+        })
+        .await?;
+    serialize_response(project, output_settings.output, output_settings.color);
+
+    Ok(())
+}
+
+pub(crate) async fn edit(
+    mut client: Client,
+    organization_id: Uuid,
+    project_id: Uuid,
+    name: String,
+    output_settings: OutputSettings,
+) -> Result<()> {
+    let project = client
+        .projects()
+        .update(&ProjectPutRequest {
+            id: project_id,
+            organization_id,
+            name,
+        })
+        .await?;
+    serialize_response(project, output_settings.output, output_settings.color);
+
+    Ok(())
+}
+
+pub(crate) async fn delete(mut client: Client, project_ids: Vec<Uuid>) -> Result<()> {
+    let count = project_ids.len();
+
+    let result = client
+        .projects()
+        .delete(ProjectsDeleteRequest { ids: project_ids })
+        .await?;
+
+    let projects_failed: Vec<(Uuid, String)> = result
+        .data
+        .into_iter()
+        .filter_map(|r| r.error.map(|e| (r.id, e)))
+        .collect();
+    let deleted_projects = count - projects_failed.len();
+
+    match deleted_projects {
+        2.. => println!("{} projects deleted successfully.", deleted_projects),
+        1 => println!("{} project deleted successfully.", deleted_projects),
+        _ => (),
+    }
+
+    match projects_failed.len() {
+        2.. => eprintln!("{} projects had errors:", projects_failed.len()),
+        1 => eprintln!("{} project had an error:", projects_failed.len()),
+        _ => (),
+    }
+
+    for project in &projects_failed {
+        eprintln!("{}: {}", project.0, project.1);
+    }
+
+    if !projects_failed.is_empty() {
+        process::exit(1);
+    }
```

Comment:
I think it would be cleaner to return an Err here and have the process exit handling in `main.rs`.

Reply:
Updated in https://github.com/bitwarden/sdk/commit/f599033914f1e8740cac3cd64f6f1c0927915500.